src/familyhubs-frontend/private/tmp/js/components/autocomplete.js

```diff
@@ -0,0 +1,10 @@
+export function initializeAutocompletes() {
+    var autocompletes = document.querySelectorAll('[data-module="fh-autocomplete"]');
```

Comment:
question: should these tmp/js files be included in the Git repo?

Reply:
Including them means debugging works out of the box (after a clone), without the developer having to run the private js gulp task, so it creates a pit of success.
The reason they're needed for debugging to work, is because the map files reference them.